src/OpenDebugAD7/MILaunchOptions.cs

```diff
@@ -292,12 +292,32 @@ private static string CreateArgumentList(IEnumerable<string> args)
                     if (stringBuilder.Length != 0)
                         stringBuilder.Append(' ');
 
-                    stringBuilder.Append(Terminal.Quote(arg));
+                    stringBuilder.Append(QuoteArgument(arg));
                 }
             }
             return stringBuilder.ToString();
         }
 
+        // gdb does not like parenthesis without being quoted
+        private static char[] s_CHARS_TO_QUOTE = { ' ', '\t', '(', ')' };
+        private static string QuoteArgument(string argument)
+        {
+            if (!string.IsNullOrWhiteSpace(argument))
+            {
+                if (argument.IndexOfAny(s_CHARS_TO_QUOTE) >= 0)
+                {
+                    // if argument starts with a quote, escape start and ending quotes
+                    if (argument[0] == '\"')
```

Comment:
Isn't this still a problem is the args is `[ "This is a string\"With quotes\" in it." ]`? Or does GDB just not have a good way of escaping quotes that might be in it?

Reply:
The user would need to escape it as "\\\"" to get it to work. we can replace all instances of "\"" with "\\\"" first and then do a trim and replace.

Reply:
I'm worried about users who have escaped it properly today being broken if we start replacing the ones in the middle of the arguments list.

Reply:
But in the json the user already did escape it. You mean that they would need to use:

`args: [ "This is a string \\\"with quotes\\\" in it." ]`

Personally I think compat is a bad reason to leave it broken (assuming we can easily fix it).

Reply:
Ok. I'll make the change and see if anyone complains.